src/evo/deterministicmns.cpp

```diff
@@ -1305,17 +1305,13 @@ static bool CheckPlatformFields(const ProTx& proTx, CValidationState& state)
         if (proTx.platformP2PPort != mainnetPlatformP2PPort) {
             return state.Invalid(ValidationInvalidReason::TX_BAD_SPECIAL, false, REJECT_INVALID, "bad-protx-platform-p2p-port");
         }
-    } else if (proTx.platformP2PPort == mainnetPlatformP2PPort || proTx.platformP2PPort < 1 || proTx.platformP2PPort > std::numeric_limits<uint16_t>::max()) {
-        return state.Invalid(ValidationInvalidReason::TX_BAD_SPECIAL, false, REJECT_INVALID, "bad-protx-platform-p2p-port");
     }
 
     static int mainnetPlatformHTTPPort = CreateChainParams(CBaseChainParams::MAIN)->GetDefaultPlatformHTTPPort();
     if (Params().NetworkIDString() == CBaseChainParams::MAIN) {
         if (proTx.platformHTTPPort != mainnetPlatformHTTPPort) {
             return state.Invalid(ValidationInvalidReason::TX_BAD_SPECIAL, false, REJECT_INVALID, "bad-protx-platform-http-port");
         }
-    } else if (proTx.platformHTTPPort == mainnetPlatformHTTPPort || proTx.platformHTTPPort < 1 || proTx.platformHTTPPort > std::numeric_limits<uint16_t>::max()) {
```

Comment:
don't we still want the check that it's not the main net httpport or nah?

Reply:
@PastaPastaPasta Nope. Platform requested to relax this constraint.